python-flask/docker-compose.yaml

```
@@ -0,0 +1,21 @@
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: flask_app
+    environment:
```

We have a .env.local in the root folder. It would be nice to use the same  one among all the applications. So I think would be a good idea to use
```suggestion
        env_file:
          - ../.env.local
```
and then change the SQLALCHEMY_DATABASE_URI to DATABASE_URL. We can add the SECRET_KEY there too then.

---

```suggestion
services:
  python-flask:
    image: waect/python-flask
    build:
      context: .
      dockerfile: Dockerfile
    container_name: python-flask
    env_file:
      - ../.env.local
    ports:
      - "5000:5000"
    networks:
      - waect-network

networks:
  waect-network:
    external: true
```

---

We also need to talk about the docker files. We need to use the same base image for it.

---

Changed the `docker-compose` file to load the env_file (currently still from the application folder, will be changed as soon as we have a .env at the top level). Changed naming of the DB URL in the `.env`file to `DATABASE_URL`.